class-components/src/App.tsx

```diff
@@ -0,0 +1,87 @@
+import { Component } from 'react';
+
+import './App.css';
+import Search from './components/Search/Search';
+
+import { IAppState, ISingleResult } from './types/SearchTypes';
+import { BASE_URL, LS_QUERY } from './constants';
+import Results from './components/Results/Results';
+
+class App extends Component<object, IAppState> {
```

Comment:
type `object` in TS is equal to type `any` on practice. Just letting you know so be careful with it :)